### ISSUE TITLE
[Sugestão] Supplies como sub-categorias 

### ISSUE THREAD

**Issue by gustavocs** [SOS-RS/frontend#160]
Alguns abrigos não cadastram os suprimentos de forma detalhada e alguns usuários também não fazem a busca de forma detalhada. Às vezes uma busca por **pão de forma** não retorna um abrigo que cadastrou a necessidade de **pão**. Existem poucas categorias, então a ideia seria dar a possibilidade de juntar supplies em grupos maiores. Dessa forma, a sugestão é adicionar uma coluna `supply_id` à tabela `supply` com um self-relation para outro `supply` para facilitar a busca.

Exemplo: "**Feijão**", "**arroz**", "**farinha**" teriam uma referência para o supply "Alimentos não perecíveis". Dessa forma, uma busca filtrando por **Feijão** retornaria **Alimentos não perecíveis** e vice-versa.

"**Gás de cozinha**" poderia ser um supply como subgrupo de outros itens como "Botijão de Xkg", "Pão" seria de quaisquer sub-tipos de pães, etc.

**Comment by andradeviniicius** [comment]
Ótima ideia, porém acredito que essa demanda seja de backend @gustavocs

**Comment by gustavocs** [comment]
> Ótima ideia, porém acredito que essa demanda seja de backend @gustavocs

Verdade, tem razão. Mas como tudo em ambas aplicações vão pra um backlog único, tanto faz. Valeu!
